### PR TITLE
T/pdf for academy

### PR DESCRIPTION
**Pull request by till**
cc @dazz @sodoku — Want to test? 

---

Installs PDF lib dependency for: easybib/researchready#5234
Fix for: DEVOPS-147

---

~~Blocked by: #1018~~

### CODE REVIEW

**Comment by dazz** [comment]
testing

File: stack-academy/recipes/role-vagrant.rb [hunk 1]
```diff
@@ -2,13 +2,33 @@
 include_recipe 'ies-mysql'
 include_recipe 'ies-mysql::dev'
 include_recipe 'ies::role-phpapp'
-include_recipe 'nginx-app::vagrant'
+
+node['vagrant']['applications'].each do |app_name, app_data|
```
Comment (till) [inline stack-academy/recipes/role-vagrant.rb]: cc @gilleyj — I think this is the culprit.

I am running the VM locally to test and I am gonna run a deployment on playground to ensure I broke nothing.

**Comment by dazz** [comment]
@till this works now, thanks :+1:

**Comment by sodoku** [comment]
Thanks, @till

**Pull request merged**